src/client/src/orangehrmClaimPlugin/pages/EditClaimEvent.vue

```diff
@@ -0,0 +1,167 @@
+        {{ $t('claim.edit_event') }}
+      </oxd-text>
+
+      <oxd-divider />
+
+      <oxd-form novalidate="true" :loading="isLoading" @submit-valid="onSave">
+        <oxd-grid :cols="2" class="orangehrm-full-width-grid">
+          <oxd-grid-item>
+            <oxd-form-row>
+              <oxd-input-field
+                v-model="claimEvent.name"
+                :label="$t('claim.event_name')"
+                disabled
+                required
+              />
+            </oxd-form-row>
+          </oxd-grid-item>
+          <oxd-grid-item class="--offset-row-2">
+            <oxd-form-row>
+              <oxd-input-field
+                v-model="claimEvent.description"
+                type="textarea"
+                :label="$t('general.description')"
+                :rules="rules.description"
+              />
+            </oxd-form-row>
+          </oxd-grid-item>
+          <oxd-grid-item class="--offset-row-3">
+            <div class="orangehrm-sm-field">
+              <oxd-text tag="p" class="orangehrm-sm-field-label">
+                {{ $t('general.active') }}
+              </oxd-text>
+              <oxd-switch-input v-model="claimEvent.status" />
+            </div>
+          </oxd-grid-item>
+        </oxd-grid>
+        <oxd-divider />
+
+        <oxd-form-actions>
+          <required-text />
+          <oxd-button
+            display-type="ghost"
+            :label="$t('general.cancel')"
+            @click="onCancel"
+          />
+          <submit-button />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import {navigate} from '@ohrm/core/util/helper/navigation';
+import {APIService} from '@/core/util/services/api.service';
+import {shouldNotExceedCharLength} from '@ohrm/core/util/validation/rules';
+import SwitchInput from '@ohrm/oxd/core/components/Input/SwitchInput';
+
+const initialClaimEvent = {
+  name: '',
+  description: '',
+  status: null,
+};
+
+export default {
+  components: {
+    'oxd-switch-input': SwitchInput,
+  },
+  props: {
+    id: {
+      type: Number,
+      required: true,
+    },
+  },
+
+  setup() {
+    const http = new APIService(
+      window.appGlobal.baseUrl,
+      'api/v2/claim/events',
+    );
+    return {
+      http,
+    };
+  },
+
+  data() {
+    return {
+      isLoading: false,
+      claimEvent: {...initialClaimEvent},
+      rules: {
+        description: [shouldNotExceedCharLength(1000)],
+      },
+    };
+  },
+
+  beforeMount() {
+    this.isLoading = true;
+    this.http
+      .get(this.id)
+      .then((response) => {
+        const {data} = response.data;
+        this.claimEvent = {...data};
+      })
+      .finally(() => {
+        this.isLoading = false;
+      });
+  },
+
+  methods: {
+    onCancel() {
+      navigate('/claim/viewEvents');
+    },
+    onSave() {
+      this.isLoading = true;
+      this.http
+        .update(this.id, {
+          name: this.claimEvent.name,
```

Comment:
```suggestion
```
No need to send name as we don't allow it to be edited

Reply:
done